.travis.yml

```diff
@@ -4,7 +4,7 @@ python:
   - "3.7"
   - "3.8"
 env:
-  - DEP=[all]
+  - DEP=[all,doc,dev]
```

Comment:
does this start three jobs? if not, shouldn't "all" be sufficient?

Reply:
now it is, because I changed the definition.